src/auth/__tests__/AuthProvider.unit.test.tsx

```diff
@@ -33,51 +40,45 @@ describe("AuthProvider", () => {
   });
 
   it("provides access to all fields in interface (sanity check)", () => {
-    expect("user"            in auth).toBe(true);
-    expect("csrfToken"       in auth).toBe(true);
-    expect("check"           in auth).toBe(true);
+    expect("user" in auth).toBe(true);
+    expect("csrfToken" in auth).toBe(true);
+    expect("check" in auth).toBe(true);
     expect("isAuthenticated" in auth).toBe(true);
-    expect("isEditor"        in auth).toBe(true);
-    expect("post"            in auth).toBe(true);
-    expect("put"             in auth).toBe(true);
-    expect("patch"           in auth).toBe(true);
-    expect("delete"          in auth).toBe(true);
-    expect("setUser"         in auth).toBe(true);
-    expect("login"           in auth).toBe(true);
-    expect("logout"          in auth).toBe(true);
```

Comment:
rip the whitespacing i spent so much time on :cry:

Reply:
Sorry, we are all powerless in the face of prettier.